packages/typespec-ts/test/commands/run.ts

```diff
@@ -0,0 +1,224 @@
+import { fileURLToPath } from "url";
```

Comment:
what's the difference between this ts file and the js file? I don't think we need both?